modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java

```diff
@@ -128,6 +129,19 @@ public PortletURL getSearchURL() {
 		return portletURL;
 	}
 
+	public List<SortItem> getSortItems() {
+		List<SortItem> sortItems = new ArrayList<>();
+
+		SortItem sortItem = new SortItem();
```

Comment:
@luca-pellizzon  Why are we not using the builder?

Reply:
Inside Commerce code we are not using the builder (yet?), so I didn't use it.
For example in getShipmentItemBulkActions() class CommerceShipmentDisplayContext.java

Reply:
We are using it in all commerce code as we migrated to portal one